app/views/admin/api/index.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="admin">
+  <h2 class="heading"><%= ts('API Tokens') %></h2>
+
+  <ul class="navigation actions" role="navigation">
+    <li><%= span_if_current ts('Create a Token'), new_admin_api_path %></li>
+  </ul>
+
+  <table>
+    <thead>
+      <tr>
+        <td>Name</td>
+        <td>Token</td>
+        <td>Created</td>
+        <td>Updated</td>
+        <td></td>
+      </tr>
+    </thead>
+    <tbody>
+    <% @api_keys.each do |api_key| %>
+      <tr>
+        <td><%= api_key.name %></td>
+        <td><%= api_key.access_token %></td>
+        <td><%= api_key.created_at %></td>
+        <td><%= api_key.updated_at %></td>
+        <td>
+          <ul class="navigation actions">
+            <li><%= link_to ts("Edit"), edit_admin_api_path(api_key) %></li>
+            <li>
+            <%= link_to ts("Delete"), admin_api_path(api_key), :confirm => 'Are you sure?', :method => :delete %>

```

Comment:
Could you make the confirmation message a bit more specific, e.g. "Are you sure you want to delete this API key?" and make it translatable? (It's always a good idea to remind people exactly what they're doing, on the off chance they read the message.) Also, I think this could more over a tab, like

```
<li>
  <%= link_to ts("Delete"), admin_api_path(api_key), :confirm => ts('Are you sure you want to delete this API key?'), :method => :delete %>
</li>
```

And to be a further pain in the ass and bearer of bad news, this is only set up to handle deletion with JavaScript enabled.